src/instanceMethods/_main.js

```diff
@@ -377,6 +377,9 @@ export function _main (userParams) {
       if (innerParams.inputClass) {
         dom.addClass(inputContainer, innerParams.inputClass)
       }
+      if (innerParams.customClass) {
```

Comment:
For **performance reasons**, this should be: 

```javascript
if (innerParams.customClass.input) 
```

If you set a breakpoint inside the `if` you will see that is called 5 times (one for each input types) **even if `customClass.input` it's undefined**. I think we should avoid that...

Reply:
the performance difference is almost nothing here, what you're suggesting will cause this edge case to fail:

![image](https://user-images.githubusercontent.com/6059356/54084944-80d63280-4340-11e9-923c-eeb1dd3630a9.png)